src/LibraryManager.Vsix/Commands/InstallLibraryCommand.cs

```diff
@@ -60,52 +59,67 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
             button.Visible = button.Enabled = false;
 
             ProjectItem item = await VsHelpers.GetSelectedItemAsync();
+            Project project = await VsHelpers.GetProjectOfSelectedItemAsync().ConfigureAwait(false);
```

Comment:
>await VsHelpers.GetProjectOfSelectedItemAsync(). [](start = 30, length = 48)

Is there a way to get this from the ProjectItem rather than calling an async VSHelper method?

Reply:
ProjectItem is going to be null when command is invoked from project scope. I want to let is pass for both the scenarios: when command is invoked from folder and project

---
In reply to: [201045767](https://github.com/aspnet/LibraryManager/pull/291#discussion_r201045767) [](ancestors = 201045767)